helm/boring-registry/values.yaml

```diff
@@ -0,0 +1,110 @@
+# Default values for boring-registry.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Global options.
+global:
+  image:
+    repository: ghcr.io/tiermobility/boring-registry
+    tag: v0.7.0
+    pullPolicy: Always
+
+nameOverride: ""
+fullnameOverride: ""
+
+server:
+  # If set to true, the server component will be created.
+  enabled: true
```

Comment:
Do we have a use-case for not enabling the server component?

Reply:
Probably not, happy to remove it :)